test/services/xccdf_report_parser_test.rb

```diff
@@ -104,6 +104,9 @@ class XCCDFReportParserTest < ActiveSupport::TestCase
         rule_names = RuleResult.where(id: rule_results.ids).map(&:rule)
                                .pluck(:ref_id)
         assert rule_names.include?(@report_parser.oscap_parser.rule_ids.sample)
+        @report_parser.oscap_parser.rule_results.each do |rule_result|
+          assert_equal rule_result.result, RuleResult.joins(:rule).find_by(rules: {ref_id: rule_result.id}).result
```

Comment:
This asserts for each xml rule_result, that the stored DB RuleResult result with matching ref_id Rule, matches the xml result.

Reply:
How's this not throwing rubocop errors at  :thinking:

Reply:
Maybe because it's in `test/`? Only class length and block length should be unlimited here.

Reply:
I just tried running rubocop locally, and this was automatically corrected. Perhaps that's why? I've started having issues with rubocop running during `rake test:validate` this week.

```
$ bundle exec rake test:validate
invalid option: --rails
... (no rubocop run, but the tests run)
```